MapboxNavigation/NavigationViewController.swift

```diff
@@ -538,6 +538,10 @@ extension NavigationViewController: RouteMapViewControllerDelegate {
         }
         return roadName
     }
+    
+    @objc public func label(_ label: InstructionLabel, willPresent instruction: VisualInstruction, as presented: NSAttributedString) -> NSAttributedString? {
```

Comment:
Curious, why does NavigationViewController need to implement the delegate method?

Reply:
This is just plumbing the delegate method through the view-controller hierarchy, just like we do [here](https://github.com/mapbox/mapbox-navigation-ios/pull/1530/files#diff-ab5a908dc637e51fa78cf2be28d033ecL527) for a `NavigationMapViewDelegate` method or [here](https://github.com/mapbox/mapbox-navigation-ios/pull/1530/files#diff-292af689ce88d5b5aa06966456abc0e9R607) for the _same_ method (`label(_:willPresent:as:)`), being routed through `RouteMapViewController`.

It's not really accurate to say that the `NavigationViewController` is _implementing_ the delegate method so much as it's merely passing the message onto _its_ delegate.